### PR TITLE
Make camel test more robust

### PR DESCRIPTION
**Pull request by OpenGuidou**
I've aligned the camel timeout to the one defined for nats, and adding a retry mechanism on the test.

Could you run the CI a few times and check if it improves the situation ?

### CODE REVIEW

File: smallrye-reactive-messaging-camel/src/test/java/io/smallrye/reactive/messaging/camel/outgoing/OutgoingCamelTest.java [hunk 1]
```diff
@@ -132,9 +133,11 @@ void testWithBeanDeclaringASimpleProcessorThatForwardIncomingHeaders() throws Ex
 
             final CompletableFuture<Exchange> completableFuture = CompletableFuture
                     .supplyAsync(
```
Comment (cescoffier) [inline smallrye-reactive-messaging-camel/src/test/java/io/smallrye/reactive/messaging/camel/outgoing/OutgoingCamelTest.java:135]: I would not use supplyAsync with a specific executor. The behavior is not the one you think when the level of parallelism is 1.

File: smallrye-reactive-messaging-camel/src/test/java/io/smallrye/reactive/messaging/camel/outgoing/OutgoingCamelTest.java [hunk 2]
```diff
@@ -132,9 +133,11 @@ void testWithBeanDeclaringASimpleProcessorThatForwardIncomingHeaders() throws Ex
...
-                            () -> camelContext().createConsumerTemplate().receive("nats:out?servers=127.0.0.1:7656&connectionTimeout=60000", 5000));
+                            () -> camelContext().createConsumerTemplate()
+                                    .receive("nats:out?servers=127.0.0.1:7656&connectionTimeout=60000", 60000));
```
Comment (cescoffier) [inline smallrye-reactive-messaging-camel/src/test/java/io/smallrye/reactive/messaging/camel/outgoing/OutgoingCamelTest.java:137]: I don't believe it's a timeout issue, but let's see.

**Review by cescoffier — CHANGES_REQUESTED** [review]

**Comment by cescoffier** [comment]
I'm going to execute the CI build a few times to be sure it fixes the issue.

**Review by cescoffier — APPROVED** [review]

**Comment by OpenGuidou** [comment]
Should I understand that it was better that before ?

**Comment by cescoffier** [comment]
I ran the CI a few times and it seems to be fine. Let's see how it is going to behave in the next few days.

**Pull request merged**